docs/bin/freezeAtVersion.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/local/bin/python3
+
+import argparse
+import fileinput
+import os
+from shutil import copyfile
+
+def updateFile(destination, version):
+    versiondots = version.replace('_', '.')
+    with fileinput.FileInput(destination, inplace=True) as file:
+        for line in file: 
+            print(line.replace("permalink:", "version: " + versiondots + "\npermalink:"), end='')
+    with fileinput.FileInput(destination, inplace=True) as file:
+        for line in file:
+            print(line.replace("/docs", "/docs/" + version), end='')
+
+def performFreeze(version):
+    directory = "current_docs"
```

Comment:
Yes and no - we need to commit it because these are the files which are served by Jekyll. Unfortunately it can't generate them itself because it's a static page server :(